Sources/Containerization/LinuxContainer.swift

```diff
@@ -62,6 +62,9 @@ public final class LinuxContainer: Container, Sendable {
         public var virtualization: Bool = false
         /// Optional destination for serial boot logs.
         public var bootLog: BootLog?
+        /// EXPERIMENTAL: Path in the root filesystem for the virtual
```

Comment:
Are the semantics such that if defined, this overrides using the default runtime offered by the VMM? It might be good to spell this out in this docc.